### DISCUSSION TITLE
exclusiveMinimum not handled correctly

### DISCUSSION THREAD

**Discussion by jeremyschiemann** [fastapi/fastapi#7408] [Questions] [answered]
### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI, Query

app = FastAPI()


@app.get('/')
def root(number: int = Query(..., gt=0)):
    return {"Hello": "World"}

```

### Description

Setting `gt=0` ([docs](https://fastapi.tiangolo.com/tutorial/path-params-numeric-validations/#number-validations-greater-than-or-equal)) produces a openapi schema with `"exclusiveMinimum": 0.0` for that parameter.
According to [openapi ](https://swagger.io/docs/specification/data-models/data-types/) (see "Minimum and Maximum") this field has to be a boolean and present together with the `minimum` field. 

Setting both, `gt` and `ge`, doesn't work although that should be the correct way. `gt` should also be a boolean instead of float.
(Most likely same for `lt`/`le`)

### Environment

* OS: Windows 10
* FastAPI Version 0.60.1
* Python version: 3.8

**Comment by rkbeatss** [comment]
Refer to #240 to track this

**Comment by tiangolo** [comment]
Thanks for the help here @rkbeatss ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:

**Comment by Kludex** [comment, marked as answer]
Fixed in version 0.100.0 with Pydantic V2 support.
